examples/tokenless/vite.config.ts

```diff
@@ -17,7 +17,6 @@ export default defineConfig({
     codecovVitePlugin({
       enableBundleAnalysis: true,
       bundleName: "@codecov/example-tokenless-app",
-      uploadToken: process.env.TOKENLESS_UPLOAD_TOKEN,
```

Comment:
or does it have to be a username?

Reply:
might be hard to fudge the `:`

to get the branch: 
first - for BA, shelter gets the `commit_id` from the `commit` field in the `body` of the request. The `commit` and `repo` have to exist in firestore - I'm thinking they will. If they exist, shelter uses the `branch` from the `commit`.

if `shelter` can't get the `commit` (if you found a way to break the above method), it will use the `branch` field in the `body` of the request.

and then we have an additional check just for BA uploads, that the `branch` (most likely from the `commit`) has to match the `branch` in the `body` of the request.

This might explain why the tokenless example doesn't work on `shelter`